docs/concepts.rst

```diff
@@ -539,13 +539,22 @@ Most Python objects can be serialized with Python's built-in `pickle
 <https://docs.python.org/3/library/pickle.html>`_ module.  However, for some
 object types it's more efficient or more idiomatic to use a different format.
 There are also some types of objects that can't be pickled at all.  Bionic uses
-``pickle`` by default, but handles some types specially: `Pandas
-<https://pandas.pydata.org/>`_ DataFrames are serialized in the `Parquet
-<https://parquet.apache.org/>`_ format, while `Pillow
-<https://pillow.readthedocs.io/en/stable/>`_ Images are serialized as `PNG
-<https://en.wikipedia.org/wiki/Portable_Network_Graphics>`_\ s.  You can
-explictly specify a serialization strategy for an entity by attaching a
-`Protocol`_ to its definition.
+``pickle`` by default, but handles some types specially:
+
+- `Dask <https://docs.dask.org/>`_ Dataframes are serialized in the `Parquet
```

Comment:
nit: It seems a little off that more common formats like Pandas come towards the end here. Maybe we could order them by roughly descending popularity?

My guess:
* Pandas
* NumPy
* Pillow
* Dask
* GeoPandas